tests/Hal/Serializer/ItemNormalizerTest.php

```diff
@@ -207,4 +213,133 @@ public function testNormalizeWithoutCache()
         ];
         $this->assertEquals($expected, $normalizer->normalize($dummy, null, ['not_serializable' => function () {}]));
     }
+
+    public function testMaxDepth()
+    {
+        $setId = function (MaxDepthDummy $dummy, int $id): void {
```

Comment:
`void` isn't supported in PHP 7.0. It's why Travis is red.

Reply:
fixed that, but there is a strange random behat failure.